src/python/pants/backend/jvm/subsystems/scala_coverage_platform.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import os
+from pants.build_graph.injectables_mixin import InjectablesMixin
+from pants.subsystem.subsystem import Subsystem
+from pants.java.jar.jar_dependency import JarDependency
+from pants.backend.jvm.targets.jar_library import JarLibrary
+from pants.build_graph.address import Address
+from typing import Any, Dict, List, Optional, Union
+
+SCOVERAGE = "scoverage"
+blacklist_file = 'new_blacklist_scoverage'
+
+
+class ScalaCoveragePlatform(InjectablesMixin, Subsystem):
```

Comment:
Naming nit: rather than calling this `ScalaCoverage`, we should probably call it `Scoverage`, as that seems to be the project's actual name.